src/main/resources/swagger.api/paths/search-consortium/search-consortium-locations.yaml

```diff
@@ -0,0 +1,23 @@
+get:
+  operationId: getConsortiumLocations
+  description: Get a list of locations (only for consortium environment)
+  tags:
+    - search-consortium
+  parameters:
+    - $ref: '../../parameters/tenant-id-query-param.yaml'
+    - $ref: '../../parameters/consortium-limit-param.yaml'
+    - $ref: '../../parameters/offset-param.yaml'
+    - $ref: '../../parameters/sort-by-location-name-param.yaml'
+    - $ref: '../../parameters/sort-order-param.yaml'
```

Comment:
What's a defaul value for this? According to task description - we should have ASC